lms/djangoapps/program_enrollments/api/api.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this file should be at the root of the program_enrollments directory, not within the `api` directory. The convention that that ADR outlines is an `api.py` at the root of the djangoapp directory. Things under an api directory are usually related to the REST API implemented by the djangoapp.

Reply:
We will need to restructure their app then :( -- they have an api directory, which prevents the import from working

Reply:
Ah, true. Nevermind, what you have here works.

Reply:
honestly, i don't like this api.api solution. this is just a thing i threw down to get things working as a first pass. i'll probably see if there's a cleaner way to do this